src/libraries/Microsoft.Win32.Registry/src/Microsoft/Win32/RegistryKey.Windows.cs

```diff
@@ -83,22 +82,20 @@ private void FlushCore()
             }
         }
 
-        private unsafe RegistryKey CreateSubKeyInternalCore(string subkey, RegistryKeyPermissionCheck permissionCheck, RegistryOptions registryOptions)
+        private unsafe RegistryKey? CreateSubKeyInternalCore(string subkey, RegistryKeyPermissionCheck permissionCheck, RegistryOptions registryOptions)
```

Comment:
Looks like the return type is annotated as nullable on account of the code path on line 121. But it seems clear by the assertion above that this is not expected to happen unless there is a bug. I think non-nullability should be asserted.

Reply:
Good point, if this not return null then all `public RegistryKey CreateSubKey(...)` overloads not return null